src/sagemaker/workflow/airflow.py

```diff
@@ -279,6 +279,57 @@ def tuning_config(tuner, inputs, job_name=None):
     return tune_config
 
 
+def update_submit_s3_uri(estimator, job_name):
+    """Updated the S3 URI of the framework source directory in given estimator.
+
+    Args:
+        estimator (sagemaker.estimator.Framework): The Framework estimator to update.
+        job_name (str): The new job name included in the submit S3 URI
+
+    Returns:
+        str: The updated S3 URI of framework source directory
+    """
+    if estimator.uploaded_code is None:
+        return
+
+    postfix = os.path.join('/', 'source', 'sourcedir.tar.gz')
```

Comment:
if we're actually concerned about using the correct separator for the OS, then you should use `os.path.sep` instead of `'/'`. however, if this is something where we actually just need slashes, then you can just use a format string here.